cli/compose/convert/service.go

```diff
@@ -675,3 +682,15 @@ func convertCredentialSpec(namespace Namespace, spec composetypes.CredentialSpec
 	}
 	return &swarmCredSpec, nil
 }
+
+func convertCapabilities(capAdd, capDrop []string, privileged bool) ([]string, error) {
```

Comment:
Can you write a test for convertCapabilities in its current state?

Reply:
Should be now